public/employers/groupon.yml

```diff
@@ -0,0 +1,43 @@
+name: Groupon, Inc.
+shortName: Groupon
+ticker: GRPN
+wiki: Groupon
+image: groupon.svg
+officialWebsite: https://press.groupon.com/about-groupon/
+industries:
+  - eCommerce
+location:
+  city: Chicago
+  state: IL
+  country: us
+  international: true
+  wiki: Chicago
+employeesBefore:
+  type: approximately
+  upperBound: 7000
+  year: 2020
+summary: Groupon implemented a number of useful cost-cutting measures, such as reducing marketing initiatives, but also resorted to furloughs while not reducing cash compensation for executive board members.
+citations:
+  - summary: Groupon let go of combined total of 2,800 workers&mdash;40% of their workforce&mdash;through layoffs and furloughs.
+    positivity: -1
+    type: publication
+    sources:
+      - source: The Motley Fool
+        link: https://web.archive.org/web/20200423022246/https://www.fool.com/investing/2020/04/15/groupon-to-lay-off-or-furlough-2800-workers-initia.aspx
+        date: 2020-04-15T13:09:00Z
+  - summary: Groupon implemented a hiring freeze, reduced marketing initiatives, and other cost-cutting measures.
```

Comment:
I think this should be a 0 as it doesn't really benefit their employees too much.

Reply:
That's fair. It may result in preventing layoffs, but since they did that anyway, maybe a 0 is fair.